### ISSUE TITLE
Quad IO without continuous mode/mode bits ...

### ISSUE THREAD

**Issue by myklemykle** [adafruit/nvm.toml#17]
Hello, I've been looking at an upstream project [pico-boot2-generator](https://github.com/maxgerhardt/pico-boot2-generator) which uses your database to auto-generate second-stage boot code for the RP2040.  I am trying to figure out ways to support the Winbond [W25Q128JW](https://www.winbond.com/resource-files/W25Q128JW_RevG_07292021%20Plus.pdf) 1.8v flash chip.  

According to the datasheet, this chip supports Fast Read Quad I/O command (0xEB), but it does not support using the Continuous Mode bits that tell the flash to stay in that command forever. So the SSI needs to issue the 0XEB command on every read. No big deal, I can see how to support that in hand-written code, but I'm interested in getting it to work with pico-boot2-generator. 

However, in all of the flash support code I've seen so far there seems to be an assumption that Continuous Mode is supported wherever Quad I/O is supported.  And yet this appears not to be the case for my chip.

Basically it seems to me that there's a feature of flash chips, Continuous Mode Supported (true/false), that's missing from your schema. If it was in the schema, then tools like pico-boot2-generator could use it to auto-generate proper code for chips like the W25Q128JW.  

But I may be wrong?  Is there some existing field in your schema that covers this?

**Comment by tannewt** [comment]
I'd be happy to merge a PR that adds a new field for that.
